Mathlib/NumberTheory/LSeries/DirichletContinuation.lean

```diff
@@ -72,6 +72,15 @@ lemma LFunction_eq_LSeries (χ : DirichletCharacter ℂ N) {s : ℂ} (hs : 1 < r
     LFunction χ s = LSeries (χ ·) s :=
   ZMod.LFunction_eq_LSeries χ hs
 
+lemma deriv_LFunction_eq_deriv_LSeries {n : ℕ} [NeZero n] (χ : DirichletCharacter ℂ n) {s : ℂ}
```

Comment:
```suggestion
lemma deriv_LFunction_eq_deriv_LSeries (χ : DirichletCharacter ℂ N) {s : ℂ}
```
There is a nonzero `N` in scope, why not use it?

Reply:
Because I forgot to check what's in scope...